app/src/main/java/com/github/bsideup/liiklus/config/GatewayConfiguration.java

```diff
@@ -46,6 +46,7 @@ public void initialize(GenericApplicationContext applicationContext) {
         applicationContext.registerBean(LiiklusService.class);
     }
 
+    @ConfigurationProperties("layers")
```

Comment:
get back to annotations?

Reply:
yeah, everything in one place, and to have less differences with the classical boot app